Volume/Networking/UserQueries.graphql

```diff
@@ -0,0 +1,21 @@
+fragment weeklyDebriefFields on WeeklyDebrief {
+    createdAt
+    expirationDate
+    numReadArticles
+    numShoutouts
+    numBookmarkedArticles
+    readArticles {
+        ...articleFields
+    }
+    randomArticles {
+        ...articleFields
+    }
+}
+
+query GetWeeklyDebrief($uuid:String!) {
+    user: getUser(uuid:$uuid) {
+        weeklyDebrief: weeklyDebrief {
+            ...weeklyDebriefFields
```

Comment:
If you only use the weeklyDebriefFields once, there's no need to separate it into a fragment. I would only use fragments if you're reusing them